370-auth-ui/oidc.md

```diff
@@ -0,0 +1,351 @@
+# OpenID Connect (OIDC) Identity Provider
+
+## Overview
+
+Skygear Auth offers authentication flow which conforms OIDC specification. You can integrate Skygear Auth as OIDC auth provider in your application or use Skygear Platform with Skygear Microservices directly.
```

Comment:
Standardize the terms.

"OIDC auth provider" -> "OpenID Provider" or "OP" (This is the term used in OIDC spec)
"auth provider" -> "OP"